todo/src/reducers/index.js

```diff
@@ -0,0 +1,38 @@
+export const ADD_TODO = "ADD_TODO";
+export const TOGGLE_COMPLETED = "TOGGLE_COMPLETED";
+
```

Comment:
these are conventionally written in a separate constants file or inside the actions file as they are the action-types strings. this works but is a bit unconventional